### PR TITLE
rearranging develop to current project progress

### PR DESCRIPTION
**Pull request by TheHarbinger99**
Hey team, have rearranged developed to current team progress. Travis green (no awseb commands), aws eb deployment green both checks passed. Please merge this and from now on any work that needs to be done should be forked from developed and added to develop once done. Also do not under any condition make changes to develop directly without doing a pull request.

### CODE REVIEW

**Review by ds6332 — APPROVED** [review]

**Review by guptaviha — CHANGES_REQUESTED** [review]
Looks good. Suggest moving the pip installs of black, flake8, coverage and coveralls in travis.yaml into requirements.txt so that everyone's local venv will not have missing packages

**Comment by TheHarbinger99** [comment]
got it , moved them to requirements.txt

File: README.md [hunk 1]
```diff
@@ -1,7 +1,3 @@
-![Travis_Build](https://img.shields.io/travis/com/gcivil-nyu-org/team-5-inperson/develop)
```
Comment (guptaviha) [inline README.md:1]: Looks like this is by mistake

File: README.md [hunk 2]
```diff
@@ -1,7 +1,3 @@
...
-
-[![Coverage Status](https://coveralls.io/repos/github/gcivil-nyu-org/team-5-inperson/badge.svg?branch=develop)](https://coveralls.io/github/gcivil-nyu-org/team-5-inperson?branch=develop)
```
Comment (guptaviha) [inline README.md:3]: Looks like this is by mistake

File: .travis.yml [hunk 3]
```diff
@@ -21,14 +19,3 @@ after_script:
 
 after_success:
   coveralls
-
-deploy:
```
Comment (guptaviha) [inline .travis.yml:25]: Is this removal intentional?
